ci/token.sh

```diff
@@ -4,6 +4,11 @@ cd "$(dirname "$0")/../token/js"
 
 set -e
 
+./do.sh update
+./do.sh build memo
+./do.sh doc memo
+./do.sh test memo
```

Comment:
You've probably already noticed the dir location at the top of this file, which is why the `./do.sh` calls are failing.
Do you think we need to test all these bash pathways as well as the js analogs below?

Reply:
Yeah, those ci changes are wip since I ran into issues with missing bin and node_modules.  I'm still seeing some issues and I'll update this pr when I have those worked out.  I do think we want to test those routes or maybe better yet remove those routes from npm since not all programs will support them.  Having those routes but not testing them are an untested source of failure over time.

Reply:
Removal from npm was the way I was leaning as well.